app/code/Magento/Multishipping/Test/Unit/Model/Cart/CartTotalRepositoryPluginTest.php

```diff
@@ -6,75 +6,133 @@
-        $shippingPrice = '10.00';
-        $quoteMock = $this->createPartialMock(
-            \Magento\Quote\Model\Cart\Totals::class,
+        $objectManager = new ObjectManager($this);
+        $this->quoteTotalsMock = $this->createPartialMock(
+            QuoteTotals::class,
             [
-            'getStore',
-            'getShippingAddress',
-            'getIsMultiShipping'
+                'getStore',
+                'getShippingAddress',
+                'getIsMultiShipping'
             ]
         );
-        $this->quoteRepositoryMock->expects($this->once())->method('getActive')->with($cartId)->willReturn($quoteMock);
-        $quoteMock->expects($this->once())->method('getIsMultiShipping')->willReturn(true);
-        $shippingAddressMock = $this->createPartialMock(
-            \Magento\Quote\Model\Quote\Address::class,
+        $this->shippingAddressMock = $this->createPartialMock(
+            QuoteAddress::class,
             [
-            'getShippingMethod',
-            'getShippingRateByCode',
-            'getShippingAmount'
+                'getShippingMethod',
+                'getShippingRateByCode',
+                'getShippingAmount'
             ]
         );
-        $quoteMock->expects($this->any())->method('getShippingAddress')->willReturn($shippingAddressMock);
-
-        $shippingAddressMock->expects($this->once())->method('getShippingMethod')->willReturn($shippingMethod);
-        $shippingAddressMock->expects($this->any())->method('getShippingAmount')->willReturn($shippingPrice);
-        $shippingRateMock = $this->createPartialMock(
-            \Magento\Quote\Model\Quote\Address\Rate::class,
+        $this->shippingRateMock = $this->createPartialMock(
+            QuoteAddressRate::class,
             [
-            'getPrice'
+                'getPrice'
             ]
         );
-        $shippingAddressMock->expects($this->once())->method('getShippingRateByCode')->willReturn($shippingRateMock);
-
-        $shippingRateMock->expects($this->once())->method('getPrice')->willReturn($shippingPrice);
-
-        $storeMock = $this->getMockBuilder(\Magento\Store\Model\Store::class)
+        $this->storeMock = $this->getMockBuilder(Store::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $quoteMock->expects($this->any())->method('getStore')->willReturn($storeMock);
-        $storeMock->expects($this->any())->method('getBaseCurrency')->willReturnSelf();
+        $this->quoteRepositoryMock = $this->createMock(CartRepositoryInterface::class);
+        $this->quoteTotalRepositoryMock = $this->createMock(CartTotalRepository::class);
+        $this->modelRepository = $objectManager->getObject(CartTotalRepositoryPlugin::class, [
+            'quoteRepository' => $this->quoteRepositoryMock
+        ]);
+    }
+
+    /**
+     * Test quotTotal from cartRepository after get($cartId) function is called
+     */
+    public function testAfterGet()
+    {
+        $cartId = '10';
```

Comment:
Instead of using local variable, it's recommended to use `private const STUB_CART_ID` - with `STUB_` prefix that means it's just the example value that does not affect test itself.